region/lowernorfair/east.json

```diff
@@ -427,36 +450,71 @@
           "to": [
             {
               "id": 1,
-              "requires": [
-                {"or": [
-                  "heatProof",
-                  "canHeatRun"
-                ]},
-                {"obstacle":{
-                  "id": "A",
-                  "requires":["canUsePowerBombs"]
-                }},
-                {"heatFrames": 350}
+              "strats": [
+                {
+                  "name": "Hjless",
+                  "notable": false,
+                  "requires": [
+                    "canNavigateHeatRooms",
+                    {"heatFrames": 400}
+                  ],
+                  "obstacles": [
+                    {
+                      "id": "A",
+                      "requires": ["canUsePowerBombs"]
+                    }
+                  ],
+                  "note": [
+                    "Destroying the obstacle isn't seen to take extra time for two reasons.",
+                    "1- It destroys the shot blocks as well which are assumed to be manually destroyed each time since they respawn.",
+                    "2- This assumes no jump assists, and some of the time loss of waiting for the PB is offset by having to setup for a pretty tight wall jump setup anyway.",
+                    "It might bo slightly faster to do a stationary spinjump rather than the setup, but we won't go into that much detail."
```

Comment:
Shoot first and bomb first looked to be the same speed.  Entering the room as a rising ball was fastest at 55 heatframes.  Stationary spinjump isn't any faster, but is easier particularly for the bomb first method.

With hjb, bomb first is ~5frames faster.

Reply:
That rising ball strat is really cool. I'm trying to keep heat counts in-room though, to stop complexity from rising even higher than it already has.